client/i18n/locales/english/intro.json

```diff
@@ -747,8 +747,8 @@
   "scientific-computing-with-python": {
     "title": "Scientific Computing with Python (Beta)",
     "intro": [
-      "Python is one of the most popular, flexible programming languages today. You can use it for everything from basic scripting to machine learning.",
-      "In the Scientific Computing with Python Certification, you'll learn Python fundamentals like variables, loops, conditionals, and functions. Then you'll quickly ramp up to complex data structures, networking, relational databases, and data visualization."
+      "The Scientific Computing with Python (Beta) curriculum will equip you with the skills to analyze and manipulate data using Python, a powerful and versatile programming language. You'll learn how to perform complex calculations using a variety of tools.",
```

Comment:
If by "data" you mean data analysis on data sets, we are not teaching that for now.
The certification covers core concepts like data structures, OOP, algorithms, regex, and string manipulation.

Reply:
You are right, that should be changed, can you live that as a comment suggestion?